uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -591,8 +614,19 @@ define([
 
         return loadStatuses.call(self)
           .then(function () {
-            initUserRole.call(self);
+            return initUserRole.call(self);
+          })
+          .then(function () {
             initOpenMode.call(self);
+
+            if (self.directiveOptions.forStaff) {
+              loadManagees.call(self);
```

Comment:
`loadManagees` returns a Promise. But here you are not using `$q.all` or `return`. Are you sure this is correct?